### ISSUE TITLE
help 

### ISSUE THREAD

**Issue by Ayush-Tibrewal** [csletten/eeg_stress_detection#1]
I tried so hard but not able to get the accuracy in this is there anything you can tell me , may be i am missing out with something

**Comment by csletten** [comment]
What kind of accuracies are you getting? And which dataset are you using?

**Comment by Ayush-Tibrewal** [comment]
Using same sam 40 dataset for stress classification. Using cnn lstm all these model and getting accuracy in 70's .

**Comment by csletten** [comment]
Are you using any filtering?

**Comment by Ayush-Tibrewal** [comment]
Yes tried on filtered data , using if ica and band filters , and also tried on feature such as time frequency features , time domain fracture etc
